### PR TITLE
Feature/header redesign

### PR DESCRIPTION
**Pull request by kirilknysh**
image

image

### CODE REVIEW

File: public/common/stylesheets/color-palette.styl [hunk 1]
```diff
@@ -17,3 +15,18 @@ dark-gold = #bf9b30
...
 fog-80 = rgba(255, 255, 255, 0.8)
+
+
+holy-white = #fff
+
+background-green-0 = #183d3d
+background-green-1 = #224b4b
+background-green-2 = #2d5d59
+background-green-3 = #43827f
+game-background = linear-gradient(to top, background-green-0 0%, background-green-1 35%, background-green-2 70%, background-green-3 100%)
```
Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: I'm not sure, should not palette contain only semantic-free color definitions?

**Comment by DaQuirm** [comment]
Banned characters' outline is not very distinct, right?

File: public/common/stylesheets/color-palette.styl (hunk 1 above)
Comment (kirilknysh) [inline public/common/stylesheets/color-palette.styl]: `game-background` could be moved to `public/common/stylesheets/common.styl`, OK?

**Comment by kirilknysh** [comment]
This PR is about header only :(

**Comment by DaQuirm** [comment]
Right, sorry 😿

File: public/common/stylesheets/color-palette.styl (hunk 1 above)
Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: What about the variables below? They are also semantic, aren't they.

Comment (kirilknysh) [inline public/common/stylesheets/color-palette.styl]: I don't have enough fantasy to give them all names :) I'll check if they are also used somewhere else.

Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: Sure, just let's keep them out of the palette. Does it make sense to you?

Comment (kirilknysh) [inline public/common/stylesheets/color-palette.styl]: sure, np

**Comment by DaQuirm** [comment]
LGTM

**Pull request merged**